src/src/openthread-core-efr32-config.h

```diff
@@ -275,16 +275,6 @@
 #define OPENTHREAD_CONFIG_EFR32_UART_TX_FLUSH_TIMEOUT_MS 500
 #endif
 
-/**
```

Comment:
@hemanth-silabs I see you've removed this. Who is supposed to set OPENTHREAD_CONFIG_PSA_CRYPTO_ENABLE?

Reply:
@suveshpratapa we don't rely on `OPENTHREAD_CONFIG_PSA_CRYPTO_ENABLE` anymore. We do need another macro, but that is to config the stack itself. Radio platform abstraction need not have it.

Reply:
And just to make sure I'm clear these "same" changes are going to be made to the GSDK version of the PAL, correct?

Reply:
@romacdon Yes. These changes will go to GSDK PAL as well. It is already present in a separate branch.